webapp/channels/src/utils/url.tsx

```diff
@@ -102,11 +102,15 @@ export function makeUrlSafe(url: string, defaultUrl = ''): string {
 }
 
 export function getScheme(url: string): string | null {
-    const match = (/([a-z0-9+.-]+):/i).exec(url);
+    const match = (/^!?([a-z0-9+.-]+):/i).exec(url);
 
     return match && match[1];
 }
 
+export function removeScheme(url = '') {
```

Comment:
Nit, I don't think we should use a default argument here. For this specific case, I'd want the caller to be in charge of null checking the URL, but more generally, if we were to want the function to be fine receiving an undefined value, I think it's better to write that as `url: string | undefined` since the intention of the code is clearer then. As-is, you could call `removeScheme()` and get back an empty string which feels very odd

Reply:
I agree, will change